masterlist.yaml

```diff
@@ -4434,6 +4440,13 @@ plugins:
       - 'GRA WRP Compatibility Patch no Brush Gun.esp'
       - 'GRA-WRP-Patch-Two_Unofficial.esp'
       - 'singleshot_retex.esp'
+      
+        - name: 'PAVE_NV_TTW.esp'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/74786' ]
+    tag: [ Graphics ]
+    after:
+      - 'fixy crap ttw.esp'
+
```

Comment:
```suggestion
```

Reply:
Yeah, we'll save that one for later, haha.